src/content/self-hosted/guides/okta/index.mdx

```diff
@@ -0,0 +1,161 @@
+
+Then, fill in the following values on the “New Web App Integration” window:
+
+1. Name: The name of your app (e.g. Okteto)
+2. Grant type: authorization code
+3. Sign In Redirect: https://okteto.{YOUR_DOMAIN}/auth/callback
+4. Assignments:  We recommend you create a dedicated group for Okteto, so you can control who can access your instance from Okta.
+
+Leave all the other values empty, and press the save button.
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/new-web-app-integration.png" alt="Create the new app integration" /></p>
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/new-web-app-integration-2.png" alt="Create the new app integration" /></p>
+
+Once the application has been created, copy the ClientID and the Client Secret. You'll need them to configure your Okteto Instance. 
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/secrets.png" alt="Copy the client ID and the Client Secret" /></p>
+
+Assign the users or groups that will have access to Okteto to the application that you just created.
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/assignment.png" alt="Assign the users to the application" /></p>
+
+By default, Okteto will use the attributes below when creating a user. 
+
+- `Name` as the display name of the user.
+- `Nickname` as the username of the user.
+- `Email` as the email.
+- `Picture URL` for the user's profile picture.
+
+Authentication will fail if `Nickname` or `Email` is missing. You can set these values at the User or the Application Assignment level. 
+
+> Please refer to the [Customize the attributes that Okteto consumes](#customize-the-attributes-that-okteto-consumes) section below on how to change the attribute mapping that Okteto will use during authentication. 
+
+If you want to pull the profile picture from Okta, you'll need to set the Picture URL” property of the user in Directory → People  → $User → Applications → $Application → Edit
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/picture.png" alt="Update the picture URL value in the application assignment" /></p>
+
+Finally, browse to https://{YOUR_TENANT}.okta.com/.well-known/openid-configuration, and copy the values of `issuer` and `authorization_endpoint`. You'll need this to finish configuration of your Okteto instance.
+
+
+<p align="center"><Image src="/docs/self-hosted/guides/okta/images/well-known.png" alt="Get the URLs for authorization" /></p>
+
+## Configure Okteto
+
+Once you have the Okta application ready, you'll need to update the configuration of your Okteto Self-Hosted instance. 
+
+First, update the auth section of your Helm configuration file (e.g. `values.yaml` ) with the values you obtained on the first step of this guide:
+
+```yaml
+auth:
+  openid:
+    enabled: true
+    clientId: $CLIENT_ID
+    clientSecret: $CLIENT_SECRET
+    endpoints:
```

Comment:
you mentioned it's recommended to use a group, I would mention how to configure the group here

Reply:
for Okta, the recommended way is to configure the groups that area allowed to authenticate in Okta, not in Okteto configuration.